namer/namer.go

```diff
@@ -246,6 +247,11 @@ func (ns *NameStrategy) Name(t *types.Type) string {
 			"Slice",
 			ns.removePrefixAndSuffix(ns.Name(t.Elem)),
 		}, ns.Suffix)
+	case types.Array:
+		name = ns.Join(ns.Prefix, []string{
+			"Array",
```

Comment:
This name needs to add the length, perhaps right here.

Reply:
Also needs to be covered by a test.

Reply:
Changes look good, I have a slight preference for "ArrayOf5Foo" or even just "Array5Foo" because it matches the go ordering of `[5]Foo`. What do you think?

Reply:
"Array5Foo" sounds good to me. Updated.